src/SourceBuild/tarball/content/patches/roslyn-analyzers/0001-Exclude-PerformanceTests-from-source-build.patch

```diff
@@ -0,0 +1,69 @@
+From c3e8b8aa4bb025e20dcf7da4c0281469c85824f5 Mon Sep 17 00:00:00 2001
+From: Michael Simons <msimons@microsoft.com>
+Date: Mon, 26 Jul 2021 15:59:32 +0000
+Subject: [PATCH] Exclude PerformanceTests from source build
+
+This addresses the following issue in the PerformanceTests:
+src/PerformanceTests/Utilities/Common/CompilationHelper.cs(126,23): error CS0534: 'CompilationHelper.OptionsProvider' does not implement inherited abstract member 'AnalyzerConfigOptionsProvider.GlobalOptions.get'
+
+This is a result of https://github.com/dotnet/roslyn/pull/44875 which introduced the new abstract member.
+
+Since the PerformanceTests are not necessary for source-build they were excluded.
+---
+ src/PerformanceTests/Tests/PerformanceTests.csproj               | 1 +
+ src/PerformanceTests/Utilities/CSharp/CSharpPerfUtilities.csproj | 1 +
+ src/PerformanceTests/Utilities/Common/CommonPerfUtilities.csproj | 1 +
+ .../Utilities/VisualBasic/VisualBasicPerfUtilities.csproj        | 1 +
+ 4 files changed, 4 insertions(+)
+
+diff --git a/src/PerformanceTests/Tests/PerformanceTests.csproj b/src/PerformanceTests/Tests/PerformanceTests.csproj
+index 5b32efdb2..c429c778a 100644
+--- a/src/PerformanceTests/Tests/PerformanceTests.csproj
++++ b/src/PerformanceTests/Tests/PerformanceTests.csproj
+@@ -14,6 +14,7 @@
+     <Optimize>true</Optimize>
+     <Configuration>Release</Configuration>
+     <IsPackable>false</IsPackable>
++    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
```

Comment:
📝 These changes are fine for roslyn-analyzers

Reply:
Yeah no need to patch these, we should just make these changes in the repo

Reply:
Agreed, I will back-port all patches into the roslyn-analyzers repo once this PR is merged.